tests/TestCase.php

```diff
@@ -48,6 +48,9 @@ protected function runMigrations()
         $permissionUserTable = include  __DIR__ . '/../database/migrations/create_permission_user_table.php.stub';
         $permissionUserTable->up();
 
+        $permissionPeerTable = include  __DIR__ . '/../database/migrations/create_permission_peer_table.php.stub';
+        $permissionPeerTable->up();
+
```

Comment:
New migration. This could introduce a breaking change, so it may be best to make this a major version change?